### ISSUE TITLE
The installed GUI wallet cannot find wallets in C:\Users\%USER%\Documents\Monero\wallets

### ISSUE THREAD

**Issue by elibroftw** [monero-project/monero-gui#3891]
This issue has been happening for over a month now, I'm not sure what I did, but the installed GUI cannot detect my wallets:
![image](https://user-images.githubusercontent.com/21298211/164914364-6e7a938f-7b84-44d0-9307-0c03e31185e2.png)
This is how it should look at shown by a portable GUI (downloaded from https://github.com/monero-project/monero-gui/issues/3840#issuecomment-1105961410)
![image](https://user-images.githubusercontent.com/21298211/164914406-f6f67d30-ab27-41cc-83ef-1f42167cf8c1.png)

**Comment by MaxQiu0108** [comment]
I see your directory is from %USER% file, which is a wildcard file, you could switch to another identified user or Administrator in your computer.

**Comment by elibroftw** [comment]
You misunderstand. That's just the title to help others find the issue. My wallet files are located in the user folder I'm logged in as.

**Comment by selsta** [comment]
@elibroftw Do you still have this issue?

**Comment by elibroftw** [comment]
Away from home until Friday so can't check till then.

**Comment by elibroftw** [comment]
Yeah I reinstalled the Monero GUI from the latest release and the issue is still present.

**Comment by selsta** [comment]
Thank you, what I don't understand is why there are no other reports about this issue, it must be something related to the local Windows setup.
